src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/best-practices/opentelemetry-best-practices-overview.mdx

```diff
@@ -13,7 +13,7 @@ redirects:
 freshnessValidatedDate: never
 ---
 
-Setting up OpenTelemetry and getting the most from it can be a challenge. To help you optimize your experience, we've created these best practice guides. If you're trying to resolve a specific problem, see our [troubleshooting guide](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-troubleshooting).
+Setting up OpenTelemetry and getting the most from it can be a challenge. To help you optimize your experience, we've created these best practice guides.
```

Comment:
```suggestion
Setting up OpenTelemetry and getting the most from it can be a challenge. To help you optimize your experience, we've created these best practice guides. If you're trying to resolve a specific problem, see our [troubleshooting topics](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/troubleshooting/collector-issues).
```